content/en/opentelemetry/guide/host_metadata.md

```diff
@@ -0,0 +1,102 @@
+---
+title: Mapping OpenTelemetry semantic conventions to Infrastructure List host information
+kind: guide
+further_reading:
+- link: "/opentelemetry/"
+  tag: "Documentation"
+  text: "OpenTelemetry Support in Datadog"
+---
+
+<div class="alert alert-warning">
+This feature is in public beta. If you have any feedback, contact <a href="https://docs.datadoghq.com/help/">Datadog support</a>.
+</div>
+
+## Overview
+
+The Datadog exporter supports sending system information about your hosts to Datadog, which you can see at the [Infrastructure List][6] tab. You can send this information in OTLP through the ['Resource' field][1] as part of any of the existing signals. This is supported under any [deployment pattern][9] including gateway deploys. 
+
+We use [OpenTelemetry semantic conventions][2] to recognize system information about your hosts. You can use the [recommended setup for host metrics][3] to send the necessary metrics and resource attributes to Datadog. Alternatively, you can manually send this information in the way that best fits your infrastructure.
+
+During the **public beta** you need to opt-in by setting the `datadog.host.use_as_metadata` resource attribute to `true` in all OTLP payloads with information relevant about hosts.
+
+## How to opt-in to this feature?
+
+Resources are used to populate the infrastructure list information if they have a [host-identifying attribute][10] and the `datadog.host.use_as_metadata` attribute set to `true`.
+
+To explicitly declare what resources should be used for metadata, add a boolean-valued resource attribute called `datadog.host.use_as_metadata` to all resources that have relevant host information.
+
+For example, to set this for all resources in metrics, traces, and logs, you can use the [transform processor][7] with the following configuration:
+
+```yaml
+processors:
+  transform:
+    metric_statements:
+      - context: resource
+        statements:
+          - set(attributes["datadog.host.use_as_metadata"], true)
+    trace_statements:
+      - context: resource
+        statements:
+          - set(attributes["datadog.host.use_as_metadata"], true)
+    log_statements:
+      - context: resource
+        statements:
+          - set(attributes["datadog.host.use_as_metadata"], true)
+```
+
+Lastly, add this processor to the `processors` list of all your pipelines.
+
+You also need to explicitly tag all your resources with a host-identifying attribute. This is done by default by the [recommended setup for host metrics][3].
+
+## What conventions are supported?
+
+The Datadog exporter supports both resource attribute-level semantic conventions as well as system metrics-level semantic conventions. Supported resource attribute semantic conventions are mainly under [the `host.` namespace][4] and [the `os.` namespace][8]. All supported system metrics-level semantic conventions are under [the `system.` namespace][5].
+
+The following general system conventions are supported:
```

Comment:
nit: Disregard if you think this is unnecessary, but this line could be made into a heading for navigation and clearer division of the tables. Same for lines 62 and 76.
```suggestion
### General system conventions
```

Reply:
Looks better now, thanks for the suggestion